rc/lsp.kak

```diff
@@ -1226,6 +1226,10 @@ define-command -hidden lsp-inlay-hints -docstring "lsp-inlay-hints: request inla
     }
 }
 
+define-command lsp-inlay-hint-apply-textedit -docstring "Apply the textedit for the inlay hint nearest the cursor" %{
```

Comment:
I suppose we could avoid adding magic behavior based on selection size,
by defining two different commands

1. lsp-inlay-hint-apply-textedit (apply nearest edit)
2. lsp-inlay-hint-apply-textedits (apply all edits)

or equivalently, do it with a flag like "-all", not sure which is better.